Sources/Utility/Path.swift

```diff
@@ -290,12 +290,12 @@ extension String {
             let end = utf8.endIndex
             while idx != end {
                 if utf8[idx] == UInt8(ascii: "/") {
-                    idx = idx.successor()
+                    utf8.formLocation(after: &idx)
                     if idx == end || utf8[idx] == UInt8(ascii: "/") {
                         return false
                     }
                 }
-                idx = idx.successor()
+                utf8.formLocation(after: &idx)
```

Comment:
`formLocation()` can perform the operation in place, and thus can be cheaper when indices are non-trivial.  In this case though, the index for a string view will eventually become a wrapped `Int`, so there would be no difference.  I can revert to the assignment if you prefer.